src/runtime_src/core/edge/skd/xrt_skd.cpp

```diff
@@ -167,7 +167,7 @@ namespace xrt {
   XCL_DRIVER_DLLESPEC
   void
   skd::run() {
-    int32_t kernel_return = 0;
+    ffi_sarg kernel_return = 0;
```

Comment:
This makes sense to match the argument type one line 215 for the function, ffi_call().
The next line 216 has the following code:
`args_from_host[1] = (uint32_t)kernel_return;`
which uses a "C-Style cast".  It should instead use a reinterpret cast:
`args_from_host[1] = reinterpret_cast<uint32_t>(kernel_return);`